Logging/RockLib.Logging.Analyzers.Vsix/source.extension.vsixmanifest

```diff
@@ -1,24 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
-  <Metadata>
-    <Identity Id="RockLib.Logging.Analyzers.e98e205e-cffc-4f2a-a33d-591822a455d7" Version="1.0" Language="en-US" Publisher="BFriesen"/>
-    <DisplayName>RockLib.Logging.Analyzers</DisplayName>
-    <Description xml:space="preserve">This is a sample diagnostic extension for the .NET Compiler Platform ("Roslyn").</Description>
-  </Metadata>
-  <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,)" />
-  </Installation>
-  <Dependencies>
-    <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
-  </Dependencies>
-  <Assets>
-    <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="RockLib.Logging.Analyzers" Path="|RockLib.Logging.Analyzers|"/>
-    <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="Project" d:ProjectName="RockLib.Logging.Analyzers" Path="|RockLib.Logging.Analyzers|"/>
-    <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="RockLib.Logging.Analyzers.CodeFixes" Path="|RockLib.Logging.Analyzers.CodeFixes|"/>
-    <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="Project" d:ProjectName="RockLib.Logging.Analyzers.CodeFixes" Path="|RockLib.Logging.Analyzers.CodeFixes|"/>
-  </Assets>
-  <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,)" DisplayName="Visual Studio core editor" />
-    <Prerequisite Id="Microsoft.VisualStudio.Component.Roslyn.LanguageServices" Version="[15.0,)" DisplayName="Roslyn Language Services" />
-  </Prerequisites>
+	<Metadata>
+		<Identity Id="RockLib.Logging.Analyzers.e98e205e-cffc-4f2a-a33d-591822a455d7" Version="1.0" Language="en-US" Publisher="BFriesen"/>
```

Comment:
Publisher field should probably be updated.

Reply:
:)

Side note: the Vsix project is never shipped, so unless we want to run the analyzers+codefixes locally in a separate VS "hive" (which is what happens when you run this) to see them in action (kind of a manual integration test), we should probably delete them. If we find value in keeping them, I propose we should create a "Scenarios" solution that contains pre-canned cases that will trigger the analyzers. I've done this on one of my personal projects, [Transpire](https://github.com/JasonBock/Transpire/tree/main/src). I'll bring this up on our next standup.

Reply:
OK, the plan is to get rid of the Vsix project, but create another one that references our analyzer project, and see if that gives us the real-time "visibility" on these analyzers+codefixes.